net/tcp_common.h

```diff
@@ -31,7 +31,7 @@ int tcp_connect_blocking_timeout(int fd, const struct sockaddr *servaddr,
 
 
 int tcp_sync_connect_fd(union sockaddr_union* src, union sockaddr_union* dst,
-                 enum sip_protos proto, struct tcp_conn_profile *prof, enum si_flags flags);
+                 enum sip_protos proto, struct tcp_conn_profile *prof, enum si_flags flags, int mark);
```

Comment:
the mark being passed in the function is not being used here, seems like it only needs to be used on setsockopt